README.md

```diff
@@ -1,20 +1,76 @@
 # ExBackend
 
-To start your Phoenix server:
+At this moment, the backend is composed of 2 parts:
+
+1. Back-end in Elixir which provide API,
+2. Front-End in Angular which consume API from back-end.
```

Comment:
Can we spell it Front-end ?

Reply:
Yes because it's really a Front-End but not as we think it. It's the Front-End of the Back-end, not a Front-End as a website display to a final user.

Reply:
I wanna just mean small/high case issue ...

Reply:
Ahhhh... :) 
It's done.